Facebook/BusinessExtension/Block/Pixel/ViewContent.php

```diff
@@ -25,6 +25,11 @@ public function getContentName() {
     }
   }
 
+  public function getContentType() {
+    $product = $this->_registry->registry('current_product');
+    return ($product->getTypeId() == \Magento\ConfigurableProduct\Model\Product\Type\Configurable::TYPE_CODE) ? 'product_group' : 'product';
+  }
+
```

Comment:
This function seems not being used, also cc @josejia  for review

Reply:
It overrides `Facebook\BusinessExtension\Block\Pixel\Common::getContentType()` and is used in the template `BusinessExtension/view/frontend/templates/pixel/view_content.phtml`

Reply:
Some additional thoughts on this:

I believe the other events (AddToCart, Purchase, etc.) always operate on _simple products_, so we don't need this check there.